### ISSUE TITLE
Subfolder functionality not working with Gmail account

### ISSUE THREAD

**Issue by jikamens** [Extended-Thunder/send-later#655]
> Just to be clear: the subfolder functionality isn't working for you even after you separate templates and drafts into separate folders?

Correct. I'll experiment with this again either tomorrow evening (CET) or on Sunday and report back.

_Originally posted by @quaqo in https://github.com/Extended-Thunder/send-later/issues/652#issuecomment-1751399634_

**Comment by quaqo** [comment]
Upon further assessing I think it might be related to [this](https://bugzilla.mozilla.org/show_bug.cgi?id=844494), but looking at your code it fails graciously (i.e. the message doesn't get moved, SL doesn't complain and it picks it up from the main draft folder).

**Comment by jikamens** [comment]
Yup. I've opened a new issue specifically about the fact that the add-on API doesn't report the issue back to the caller -- https://bugzilla.mozilla.org/show_bug.cgi?id=1857777.

As a workaround you can configure a different Drafts folder for TB to use for your Gmail account, instead of [Gmail]/Drafts. Otherwise I don't think there's much I can do about this.

**Comment by jikamens** [comment]
I will document this in the user guide.
